pyfar/signals/signals.py

```diff
@@ -0,0 +1,568 @@
+    rms : double, array like, optional
+        The route mean square (RMS) value of the noise signal. A multi channel
+        noise signal is generated if an array of RMS values is passed.
+        The default is 1.
+    sampling_rate : int, optional
+        The sampling rate in Hz. The default is 44100.
+    seed : int, None
+        The seed for the random generator. Pass a seed to obtain identical
+        results for multiple calls of white noise. The default is None, which
+        will yield different results with every call.
+
+    Returns
+    -------
+    signal : Signal
+        The noise as a Signal object. The Signal is in the time domain and
+        has the 'rms' FFT normalization (see pyfar.fft.normalization).
+    """
+
+    # generate the noise
+    rms = np.atleast_1d(rms)
+    noise = _generate_white_noise(n_samples, rms, seed)
+
+    # level the noise
+    noise = _normalize_rms(noise, rms)
+
+    # save to Signal
+    nl = "\n"  # required as variable because f-strings cannot contain "\"
+    comment = f"White noise signal (rms = {str(rms).replace(nl, ',')})"
+
+    signal = Signal(noise, sampling_rate, fft_norm="rms", comment=comment)
+
+    return signal
+
+
+def pink_noise(n_samples, rms=1, sampling_rate=44100, seed=None):
+    """Generate single or multi channel normally distributed pink noise.
+
+    The pink noise is generated by applying a sqrt(1/f) filter to the spectrum.
+
+    Parameters
+    ----------
+    n_samples : int
+        The length of the signal in samples
+    rms : double, array like, optional
+        The route mean square (RMS) value of the noise signal. A multi channel
+        noise signal is generated if an array of RMS values is passed.
+        The default is 1.
+    sampling_rate : int, optional
+        The sampling rate in Hz. The default is 44100.
+    seed : int, None
+        The seed for the random generator. Pass a seed to obtain identical
+        results for multiple calls of white noise. The default is None, which
+        will yield different results with every call.
+
+    Returns
+    -------
+    signal : Signal
+        The noise as a Signal object. The Signal is in the time domain and
+        has the 'rms' FFT normalization (see pyfar.fft.normalization).
+    """
+
+    # generate the noise
+    rms = np.atleast_1d(rms)
+    noise = _generate_white_noise(n_samples, rms, seed)
+
+    # apply 1/f filter in the frequency domain
+    noise = fft.rfft(noise, n_samples, sampling_rate, 'none')
+    noise /= np.sqrt(np.arange(1, noise.shape[-1]+1))
+    noise = fft.irfft(noise, n_samples, sampling_rate, 'none')
+
+    # level the noise
+    noise = _normalize_rms(noise, rms)
+
+    # save to Signal
+    nl = "\n"  # required as variable because f-strings cannot contain "\"
+    comment = f"Pink noise signal (rms = {str(rms).replace(nl, ',')})"
+
+    signal = Signal(noise, sampling_rate, fft_norm="rms", comment=comment)
+
+    return signal
```

Comment:
Should we merge those functions into `noise`? This would be more consistent with `pulsed_noise` and reduce redundancy.